server/client.go

```diff
@@ -217,17 +217,17 @@ type outbound struct {
 	p   []byte        // Primary write buffer
 	s   []byte        // Secondary for use post flush
 	nb  net.Buffers   // net.Buffers for writev IO
-	sz  int32         // limit size per []byte, uses variable BufSize constants, start, min, max.
-	sws int32         // Number of short writes, used for dynamic resizing.
-	pb  int32         // Total pending/queued bytes.
-	pm  int32         // Total pending/queued messages.
+	sz  int64         // limit size per []byte, uses variable BufSize constants, start, min, max.
+	pb  int64         // Total pending/queued bytes.
+	mp  int64         // Snapshot of max pending for client.
+	lwb int64         // Last byte size of Write.
```

Comment:
This one as well.

Reply:
If you can allow a write to be more than int32, then how this can stay a int32?

Reply:
because we limit a write to int32?

Reply:
the sz will be int32, it has a max which we do not exceed. I doubt lwb would ever be over it but we could put in a check to make sure.

Reply:
Ok, changed just pb and mp to int64 (but moved some fields to have more compact structure).